R/api-functions.R

```diff
@@ -121,8 +144,13 @@ evaluate_edi_package <- function(user_id, password, eml_file_path, environment =
 #'                             eml_file_path = "data/edi20.1.xml")}
 #' @export   
 
```

Comment:
I did some testing using the api-update branch. I was able to upload 1239.1 to the staging environment but the `upload_edi_package` returned the following message: `No encoding supplied: defaulting to UTF-8.
Error in EMLaide::upload_edi_package(user_id = Sys.getenv("EDI_USER_ID"),  : 
  RequestErrorRequest failed, check that you inputs are all valid and try again. 
          See more information on request status below: create_168550079871746301`

I checked on the staging web portal to see if it uploaded even though I got this error and saw that it was uploaded.

Reply:
I am also getting this errror for both upload and evaluating.